README.md

```diff
@@ -12,126 +12,126 @@ Before one can run or deploy, one needs to do two things:
 
 1. Ensure `nypl-digital-dev` and `nypl-sandbox` profiles are registered in `~/.aws/credentials` and `~/.aws/config`
 2. Obtain environment secrets from a colleague to initialize an environment file as follows:
-  * `cp config/qa-sample.env config/qa.env`
-  * `cp config/production-sample.env config/production.env`
+  * `cp config/sample.env config/qa.env`
+  * `cp config/sample.env config/production.env`
   * Fill in missing secrets in both environment files
 
-Note, the `DISCOVERY_STORE_CONNECTION_URI` variable should be encrypted. If you have the plaintext connection string, you can encrypt it using the aws cli:
+Note, the `DISCOVERY_STORE_CONNECTION_URI` variable should be encrypted. See [NYPL notes on encrypting secrets on the command line](https://github.com/NYPL/engineering-general/blob/master/security/secrets.md#using-the-aws-command-line) for more information.
 
-```
-aws kms encrypt --key-id "[arn for 'lambda-rds' key]" --plaintext "[plaintext connection string]"
-```
+### Verifying your configuration
+
+It can be useful to inspect the configuration represented by different env files - as a sanity check before running a command. The following displays the decrypted configuration from the given env file:
+
+`node jobs/init check --envfile [path to env file] --profile [aws profile]`
 
 ### Creating a Test Event
 
-Run this to generate `event.bibs.json` full of encoded bibs:
+Test event json files can be generated to emulate lambda invocations locally.
+
+Run this to generate `event.json` full of encoded bibs:
 
 `node kinesify-data.js event.unencoded.bibs.json event.json https://platform.nypl.org/api/v0.1/current-schemas/Bib`
 
-Or run this to generate `event.items.json` full of encoded items:
+Or run this to generate `event.json` full of encoded items:
 
 `node kinesify-data.js event.unencoded.items.json event.json https://platform.nypl.org/api/v0.1/current-schemas/Item`
 
 Alternatively, to generate a event.json from a plain marcinjson document (such as retrieved from the sierra api), you can kinesify that directly:
 
 `node kinesify-data test/data/bib-10011745.json event.json  https://platform.nypl.org/api/v0.1/current-schemas/Bib`
```

Comment:
Would like to see flags included in README, thought seeing usage suggestions on the command line is very helpful.
`node kinesify-data test/data/bib-10011745.json event.json  https://platform.nypl.org/api/v0.1/current-schemas/Bib --profile [aws profile] --envfile [path to env file]`